File: katainception-python/test/example/BankAccountShould.py
import unittest
import example


class BankAccountShould(unittest.TestCase):
    def have_balance_of_zero_when_created(self):
        bankAccount = example.BankAccount()

        self.assertEqual(bankAccount.accountValue, 0)

    def have_the_balance_increased_after_a_deposit(self):
        bankAccount = example.BankAccount()

        bankAccount.deposit(self, 10)

        self.assertEqual(bankAccount.accountValue, 10)

File: katainception-kotlin/src/test/kotlin/example/BankAccountShould.kt
package example

import org.junit.Assert.assertEquals
import org.junit.Test

class BankAccountShould {

    @Test
    fun `have balance of zero when created`() {

        val bankAccount = BankAccount()

        assertEquals(0, bankAccount.balance)

    }

    @Test
    fun `have the balance increased after a deposit`() {

        // given
        val bankAccount = BankAccount()

        // when
        bankAccount.deposit(10)

        // then
        assertEquals(10, bankAccount.balance)

    }

}